c2rust-analyze/src/equiv.rs

```diff
@@ -141,9 +143,9 @@ impl<'g> EquivSet<'g> {
 
     fn set_parent(&self, x: PointerId, parent: PointerId) {
         // Local items can point to global ones, but not vice versa.
-        if x.is_global() {
-            debug_assert!(parent.is_global());
-        }
+        //if x.is_global() {
```

Comment:
Do you have a fix for this? You could just delete it.

Reply:
Fixed in b68e56876